eo-maven-plugin/src/main/java/org/eolang/maven/name/OnDefault.java

```diff
@@ -87,7 +106,7 @@ public String toString() {
      * @return Split object to name and hash.
      */
     private String[] split() {
-        String[] splt = this.object.split(OnVersioned.DELIMITER);
+        String[] splt = this.object.value().split(OnVersioned.DELIMITER);
```

Comment:
@maxonfjvipon `OnDefault` and `OnVersioned` looks rather similar, maybe we can refactor them somehow?

Reply:
@volodya-lombrozo I don't think so, `OnVersioned` replaces tag in given object with hash, `OnDefault` checks if given object contains version and if not - put the default one. I think at the step they should stay separately

Reply:
@volodya-lombrozo here I just made `OnDefault` a bit more lazy

Reply:
@maxonfjvipon What about names? Both seems misleading to me:
1. `OnVersioned`: "replaces tag in given object with hash"
2. `OnDefault`:  "checks if given object contains version and if not - put the default one"

I agree, that they have to stay separate, but names of the objects don't match with their behaviour. Are they?

Reply:
@volodya-lombrozo yes, make sense. What about `OnVersioned` -> `OnReplaced`, `OnDefault` -> `OnVersioned`

Reply:
@maxonfjvipon Looks good. Let's do it in a separate puzzle?

Reply:
@volodya-lombrozo sure